docs/library/array.rst

```diff
@@ -1,7 +1,7 @@
-:mod:`array` -- arrays of numeric data
-======================================
+:mod:`uarray` -- arrays of numeric data
+=======================================
 
-.. module:: array
+.. module:: uarray
```

Comment:
After reviewing the rest, we are still using `MP_QSTR_array`, right? If that is not already guarded by `CIRCUITPY`, then I guess it should, since all the tests changed `uarray` to `array`. But I guess you must have dealth with that.

Reply:
Ya, it's array in objmodule.c.